Scripts/Core/Contexts/EditingContextManager.cs

```diff
@@ -303,21 +348,52 @@ internal static EditingContextManagerSettings LoadUserSettings()
 
         internal static void ResetProjectSettings()
         {
+#if UNITY_EDITOR
             File.Delete(k_UserSettingsPath);
 
             if (EditorUtility.DisplayDialog("Delete Project Settings?", "Would you like to remove the project-wide settings, too?", "Yes", "No"))
                 File.Delete(k_SettingsPath);
+#endif
         }
 
         internal static void SaveProjectSettings(EditingContextManagerSettings settings)
         {
+#if UNITY_EDITOR
             File.WriteAllText(k_SettingsPath, JsonUtility.ToJson(settings, true));
+#endif
         }
 
         internal static void SaveUserSettings(EditingContextManagerSettings settings)
         {
+#if UNITY_EDITOR
             File.WriteAllText(k_UserSettingsPath, JsonUtility.ToJson(settings, true));
+#endif
+        }
+
+#if UNITY_EDITOR
+        [ContextMenu("Utility")]
```

Comment:
I'm not sure if this should be user-facing or was just a utility function for debugging input?

Reply:
This can be removed.

Reply:
Yeah, this should be deleted